### PR TITLE
412 add images and small style fixes

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/eCcFc4Es/412-add-images-and-small-style-fixes

## Description

- Adds different images to front page
- Small fixes to other elements

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Keys.elm [hunk 1]
```diff
@@ -40,26 +40,32 @@ type Key
       -- Not Alone page
     | NotAloneTitle
     | Journey1Teaser
+    | Journey1Image
```
Comment (katjam) [inline src/Copy/Keys.elm:43]:
```suggestion
    | Journey1ImageSrc
```

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -120,6 +120,9 @@ t key =
         Journey1Teaser ->
             "He threw my phone on the floor and smashed\u{00A0}it"
 
+        Journey1Image ->
+            "/sea-map/Person_1.svg"
```
Comment (katjam) [inline src/Copy/Text.elm:124]: I guess the alts might need translating (if we have them) but the images path strings could go inline to the JourneyCard record?

File: src/EmergencyContent.elm [hunk 3]
```diff
@@ -76,6 +76,7 @@ exitButtonStyle =
         , textDecoration none
         , top (rem 5)
         , width (rem 3.75)
+        , zIndex (int 2)
```
Comment (katjam) [inline src/EmergencyContent.elm:79]: Whoop!

File: src/Page/NotAlone.elm [hunk 4]
```diff
@@ -46,6 +46,7 @@ journeyIsRevealed model journeyCard =
 
 type alias JourneyContent =
     { teaser : Key
+    , image : Key
```
Comment (katjam) [inline src/Page/NotAlone.elm:49]: `src` and could be String unless we figure out how to Elm Asset them.

File: src/View/NotAlone.elm [hunk 5]
```diff
@@ -79,9 +79,9 @@ renderRevealedCard journeyCardPosition =
             journeyContentFromCardPosition journeyCardPosition
     in
     li [ css [ cardStyle, openStyle ] ]
-        [ button [ css [ closeJourneyButtonStyle ], onClick (ToggleJourney journeyCardPosition) ] [ img [ css [ height (px 44), margin auto ], src "Close.svg", alt "Close" ] [] ]
-        , div []
-            [ p [ css [ quoteStyle ] ] [ text (t journeyContent.relatable) ]
+        [ div []
+            [ button [ css [ closeJourneyButtonStyle ], onClick (ToggleJourney journeyCardPosition) ] [ img [ css [ height (px 44), margin auto ], src "/sea-map/Close.svg", alt "Close" ] [] ]
```
Comment (katjam) [inline src/View/NotAlone.elm]: Shall we break this onto multiline?

File: src/View/NotAlone.elm [hunk 6]
```diff
@@ -124,8 +124,7 @@ cardStyle =
 closedStyle : Style
 closedStyle =
     batch
-        [ backgroundImage (url "temp.png")
```
Comment (katjam) [inline src/View/NotAlone.elm:127]: Yay!

File: src/images/sea-map/Person_6.svg [hunk 7]
```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 326.75 639.25">
```
Comment (katjam) [inline src/images/sea-map/Person_6.svg:1]: Why is this one not minified?

**Review by katjam — DISMISSED** [review]
Nothing that 'requires' action. Only thoughts.

File: src/Copy/Text.elm (hunk 2 above)
Comment (Nikomus) [reply src/Copy/Text.elm:124]: I thought they may want to shuffle order later if they were here.

File: src/images/sea-map/Person_6.svg (hunk 7 above)
Comment (Nikomus) [reply src/images/sea-map/Person_6.svg:1]: I probably pressed save in here, it's just formatted the lines out. I'm happy to leave as I'll be splitting the SVGs into layers next.

**Review by katjam — APPROVED** [review]
Merge away. These changes look exciting! :balloon:

**Pull request merged**